modules/public/technical_indicators.py

```diff
@@ -12,24 +15,27 @@ def absolute_strength_histogram(dataframe, length=9, smooth=3, mode="RSI"):
     :param mode: Indicator method. Choose from: ["RSI", "STOCHASTIC", "ADX"]
     :return:
     """
-    df = dataframe.copy()
+    try:
```

Comment:
```suggestion
    df = None
    try:
```

If you're putting the first assignment of the dataframe variable in a try function, all code editors will complain further on down the line that the 'Local variable 'df' might be referenced before assignment'